packages/mask/src/plugins/IdeaMarket/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const SUBGRAPH_URL = 'https://subgraph.backend.ideamarket.io/subgraphs/name/Ideamarket/Ideamarket/'
+export const TESTNET_SUBGRAPH_URL =
+    'https://subgraph-test-avm-l2.backend.ideamarket.io/subgraphs/name/Ideamarket/IdeamarketTESTAVML2/'
+
+export const BASE_URL = 'https://ideamarket.io'
+export const TWITTER_BASE_URL = 'https://twitter.com'
+export const ToS = 'https://docs.ideamarket.io/legal/terms-of-service'
+
+export const ideaMarketHostnames = ['ideamarket.io']
+export const ideaMarketPathnameRegexMatcher = /^\/i\/(\S+)\/(\S+)/
+
+export const TWITTER_BEARER_TOKEN =
+    'AAAAAAAAAAAAAAAAAAAAAIakawEAAAAAFtDCZS2%2Fv8a8UsU3%2Bq1PdZYCY%2Bo%3DJNGDHi0EqYR11jkW9bhU27h0tXOq52BPaEYqJlA18wUBL5rydc'
```

Comment:
```suggestion
/* cspell:disable-next-line */
export const TWITTER_BEARER_TOKEN = 'AAAAAAAAAAAAAAAAAAAAAIakawEAAAAAFtDCZS2%2Fv8a8UsU3%2Bq1PdZYCY%2Bo%3DJNGDHi0EqYR11jkW9bhU27h0tXOq52BPaEYqJlA18wUBL5rydc'
```

Reply:
btw, this bearer token is actually from my own personal twitter API account, registered with my personal infos. Would be great to have one for mask, intented for mask's plugins whom need twitter api.

Reply:
Please check `packages/web3-providers/src/twitter/index.ts`. You can have the bearer token of the current login Twitter account and add a new API for this PR.

Reply:
Eventually I can let it this way, doesn't bother me.

Reply:
Tried but apparently i cannot: "Sorry, official Twitter clients cannot use this API. Register your own."

Reply:
Solved.